docs/index.rst

```diff
@@ -53,7 +53,9 @@
    notification-routing/routing-by-namespace
    notification-routing/routing-by-type
    notification-routing/routing-exclusion
+   notification-routing/routing-by-time
```

Comment:
Can you move it up one higher in the list, as it conceptually is very similar to routing by namespace and type (this is routing by time)